### PR TITLE
Sharp Mop 2.0

### PR DESCRIPTION
**Pull request by UwUPlusUwU**
Adds sharp mop and recipe- now rebalanced!

## About The Pull Request

Adds a rebalanced sharp mop and a recipe. Edit: Shitfuck my Git pushed my other shitcode.

## Why It's Good For The Game

Mostly a proof-of-concept, but janitors can now convert their trusty brooms to a slightly better version, allowing for the dream of combat janitors. Edit: KONO DIO DA!

## Changelog
:cl:
add: Added sharpened mop.
add: Dio wiz reskins.
/:cl:

### CODE REVIEW

**Comment by UwUPlusUwU** [comment]
Haha yes, how do I stop it upstreaming help aaaaaaaaa.

**Comment by UwUPlusUwU** [comment]
Ignore the third check, it just says it was from a previous build which passed.

**Comment by Oshibkka** [comment]
sounds epic

**Comment by ondrej008** [comment]
Rename this PR to "Adds Sharp Mop and Dio emote"

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -703,3 +703,11 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 	always_availible = TRUE
+
+/datum/crafting_recipe/sharpmop
+	name = "Sharpened Mop"
+	result = /obj/item/tank/jetpack/improvised
+	time = 30
+	reqs = list(/obj/item/tank/internals/oxygen/red = 2)
```
Comment (ondrej008) [inline code/modules/crafting/recipes.dm:711]: What the fuck?

File: code/modules/opo/jojo.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
+//oneplus' shitcode
+	/obj/item/clothing/suit/space/hardsuit/shielded/wizard/dio
+	name = "strange suit"
+	desc = "Makes you fe- KONO DIO DA!"
+	icon = 'code/modules/opo/jojo.dmi'
+	icon_state = "diosuit_on_mob"
+	item_state = "diosuit_on_mob"
+
```
Comment (ondrej008) [inline code/modules/opo/jojo.dm]: At least say you added this shit.

File: beestation.dme [hunk 3]
```diff
@@ -2129,7 +2129,6 @@
 #include "code\modules\mob\living\carbon\human\say.dm"
 #include "code\modules\mob\living\carbon\human\species.dm"
 #include "code\modules\mob\living\carbon\human\status_procs.dm"
-#include "code\modules\mob\living\carbon\human\update_icons.dm"
```
Comment (ondrej008) [inline beestation.dme:2132]: ?

**Review by ondrej008 — CHANGES_REQUESTED** [review]

**Comment by qwertyquerty** [comment]
Try again

**Pull request closed without merging**